scriptmodules/emulators/reicast.sh

```diff
@@ -40,13 +40,25 @@ function install_reicast() {
 function configure_reicast() {
     mkRomDir "dreamcast"
 
-    # create bios dir. Copy dc_boot.bin and dc_flash.bin there.
-    mkdir $md_inst/data
+    # Create home VMU, cfg, and data folders. Copy dc_boot.bin and dc_flash.bin to the ~/.reicast/data/ folder.
+    mkdir -p /home/pi/.reicast/data
 
     cat > $md_inst/reicast.sh << _EOF_
 #!/bin/bash
 pushd $md_inst
-sudo aoss ./reicast.elf -config config:image="\$1"
+echo Reading the entire Reicast emulator into memory to execute from there...
+sudo mkdir tmpfs
+#TODO: Find optimal smaller tmpfs size, I do not believe anywhere near this much is required.  I have only ever seen 54MB utilized during a game of Rush 2049.
+sudo mount -o size=450M -t tmpfs none tmpfs/
+sudo cp -v * tmpfs/
+cd tmpfs
+sudo aoss ./reicast.elf -config config:homedir=/home/pi -config config:image="$1"
+cd ..
+echo Ensuring any freshly-created VMUs are owned by pi and not root...
+sudo chown -R pi:pi .reicast
+echo Freeing up memory...
+sudo umount /opt/retropie/emulators/reicast/tmpfs
+sudo rm -rf tmpfs
```

Comment:
Does this need to be absolutely pathed ? if so, you can use `$rootdir` but the mount didn't so the umount probably doesn't too if in the right folder.